examples/run_k_grid.py

```diff
@@ -22,5 +23,14 @@ def test_run_k_grid():
     k_grid111 = get_k_grid(slab111, sampling_density, verbose=True, simple_reciprocal_space_parameters=False)
     assert k_grid111 == (7, 7, 1)
 
```

Comment:
Just a minor inconsistency. I was hoping both the simple method and strict definition method would lead to same k_grid. 
I think this has to do with the conversion of sampling density to k_grid_density on line 247 `k_grid_density = 1 / (sampling_density * 2 * math.pi)` . This relationship might not be exactly correct. However, I tried to derive that myself on paper and it's not a straightforward derivation :)

Reply:
I have no further comments and everything seems good. Happy days!!

Reply:
The simple method and strict definition method will lead to same k_grid if the cell is orthogonal. In this case, a fcc111 slab is not orthogonal, the reciprocal lattice parameters in x and y dimension is larger with the strict definition than the simplified one. Therefore, the k grid with the strict definition is bigger in this case. Only minor difference though.